redesign-project/src/_includes/_nav.html

```diff
@@ -0,0 +1,20 @@
+<nav class="wrapper-site-nav">
+    <button id="js-nav-trigger" aria-controls="js-site-nav" aria-expanded="false" class="u-button-reset site-nav__trigger"
+        type="button">
+        <span aria-hidden="true" class="menu-icon">
+            <div class="menu-icon__line"></div>
+            <div class="menu-icon__line"></div>
+            <div class="menu-icon__line"></div>
+        </span>
+
+    </button>
+    <ul id="js-site-nav" class="u-list-reset site-nav__list" aria-hidden="false" data-visible="false">
+        <li class="menu_nav"><a>food</a></li>
```

Comment:
Some browsers treat `a` tags in a funny way if they don't actually link somewhere. In this or any case where you don't actually have a link ready, you can use `href="#"` 👍 
```suggestion
         <li class="menu_nav"><a href="#">food</a></li>
```